test/NuGet.Core.Tests/NuGet.Frameworks.Test/NuGetFrameworkParseTests.cs

```diff
@@ -248,14 +235,19 @@ public void NuGetFramework_ParseToShortName(string expected, string fullName)
         }
 
         [Theory]
+        // Net5.0 ERA
+        [InlineData("net5.0", ".NETFramework,Version=v5.0")]
+        [InlineData("net10.1.2.3", ".NETFramework,Version=v10.1.2.3")]
+        [InlineData("netcoreapp5.0", ".NETFramework,Version=v5.0")]
```

Comment:
Aren't these supposed to be .NETCoreApp?

Reply:
https://github.com/dotnet/designs/blame/c742ac0a589d57254926c003698c2b6652e60fdc/accepted/2020/net5/net5.md#L223

Reply:
leaving the same for now, but leaving this feedback unresolved while i play a bit with it.

Reply:
yes, going forward with .NETCoreApp as per spec for now...as we discuss options and ramifications. fix commited.